### DISCUSSION TITLE
[FR] Add the ability to detect live preview mode, when a user navigates to a different entry from the preview

### DISCUSSION THREAD

**Discussion by rishifter** [craftcms/cms#15447] [Ideas] [answered]
We show an edit button on the front-end for logged-in users with edit permissions. When showing the live preview for an entry, we disable the edit button by detecting that the entry is opened in live preview mode. However, when the user navigates to different parts of the site from within the preview mode, this detection fails. This leads to the user being able to edit a post from within the preview infinitely!

To avoid cases like these, please consider adding the ability to detect live preview mode in all entries, not just the current one which is being edited. 

Edit: Updated screen recording & language clarification. 

https://github.com/user-attachments/assets/ff10f58f-7748-4dbb-a05d-1fcdb690be10

**Comment by brandonkelly** [comment, marked as answer]
Thanks for pointing that out! I’ve just resolved this for Craft 4.11 and 5.3 (72d1d57e69edc9c1368ba7f24f02a012aad4edac).

**Reply by rishifter** [reply]
Thanks for the prompt fix!
